src/plugins/embeddable/public/lib/test_samples/embeddables/filterable_embeddable.tsx

```diff
@@ -19,12 +19,18 @@
 
 import { IContainer } from '../../containers';
 import { EmbeddableOutput, EmbeddableInput, Embeddable } from '../../embeddables';
-import { Filter } from '../../../../../data/public';
+
+/** @internal */
+export interface MockFilter {
+  $state?: any;
+  meta: any;
+  query?: any;
+}
```

Comment:
It doesn't make sense for embeddables to know about filter in the first place, but since all of these test helpers existed, I went ahead and made a "fake" filter interface for use in the tests.